toolkit/README.md

```diff
@@ -3,7 +3,7 @@
 - **Recommended if you want to try CBL-Mariner.**
 - Quickly build the **Standard CBL-Mariner** image, install and use.
 
-## [Tutorial for Building Custom Mariner Based Images](https://github.com/microsoft/CBL-MarinerDemo)
+## [Tutorial for Building Custom Mariner Based Images](https://github.com/microsoft/CBL-MarinerTutorials)
```

Comment:
Agree with these changes, but the changes need to go to the Mariner "main" branch and an equivalent change probably should be made to the "1.0-dev" branch.

Both are "staging" branches.  Any changes directly to the official release branches (2.0 and 1.0) can get "stomped"